src/Uno.UWP/UI/Core/Internal/KeyboardStateTracker.cs

```diff
@@ -95,4 +96,14 @@ private static void SetStateOnNonSideKeys(VirtualKey key)
 			_keyStates[VirtualKey.Menu] = _keyStates[key];
 		}
 	}
+
+	internal static void Reset()
+	{
+		// Clearing state when debugger is attached would
+		// make it hard to debug key events.
+		if (!System.Diagnostics.Debugger.IsAttached)
+		{
+			_keyStates.Clear();
+		}
```

Comment:
Not sure whether to go with `Debugger.IsAttached` or just `#if DEBUG` here. My thought is that if someone is debugging an Uno app project, this will make it easier to debug key-related scenarios as opposed to `#if` approach

Reply:
I think `Debugger.IsAttached` is more suitable, as the debugger is really what will cause the window to be deactivated.